clusterman/migration/worker.py

```diff
@@ -259,9 +261,18 @@ def event_migration_worker(migration_event: MigrationEvent, worker_setup: Worker
             nodes = manager.get_node_metadatas(AWS_RUNNING_STATES)
             offset = worker_setup.prescaling.of(len(nodes))
             logger.info(f"Applying pre-scaling of {offset} node to {migration_event.cluster}:{migration_event.pool}")
-            avg_weight = mean(node.instance.weight for node in nodes)
-            prescaled_capacity = round(manager.target_capacity + (offset * avg_weight))
-            manager.modify_target_capacity(prescaled_capacity)
+            capacity_offset = offset * mean(node.instance.weight for node in nodes)
+            if worker_setup.disable_autoscaling:
```

Comment:
Do we still need `disable_autoscaling` setting? Because it actually doesn't disable autoscaling.

Reply:
It does on line 252. Mostly keeping it for smoother transition since we have a few thing configured that way, so
* disable autoscaling + prescaling does the old thing
* just prescaling uses the new logic

Then I can work on removing the settings gradually.

Reply:
oops, Actually I asked question on line 252 :)
Cool, I loved idea.